api/openapi.yaml

```diff
@@ -1576,6 +1575,17 @@ components:
           # on the client side.
           #
           # readOnly: true
+        name:
+          type: string
+        scanFamiliesConfig:
+          $ref: '#/components/schemas/ScanFamiliesConfig'
+        maxParallelScanners:
+          type: 'integer'
+          minimum: 1
+          maximum: 20
+          description: "The maximum number of scanners that can run in parallel for each scan"
+        scannerInstanceCreationConfig:
+          $ref: '#/components/schemas/ScannerInstanceCreationConfig'
```

Comment:
Why `TargetScanResult` needs this info?

Reply:
Think of TargetScanResult as "AssetScan" its a scan of a specific asset and can be reconciled and created separately. i.e. like a pod. This is part of the configuration for how to run that scan.

Reply:
right now the concurrency is handled by the reconciler for a specific AssetScan, it looks to see if there are other AssetScans with the same Scan ID, and otherwise it won't schedule. Its something I want to revisit.

Reply:
Not sure I understand, can you point me to the code?
AFAIK the `maxParallelScanners` is used by the job manager to handle the max number of workers that spin jobs.